grafana-cloud-metrics.yaml

```diff
@@ -224,11 +221,36 @@ resources:
               tenant: rax_tenant
               username: rax_username
               apikey: rax_apikey
-              authentication_module: rax_auth
+              authentication_module: blueflood_graphite_finder.rax_auth
               authentication_class: BluefloodAuth
               urls:
                 - http://iad.metrics.api.rackspacecloud.com
+            logging:
+              version: 1
+              disable_existing_loggers: true
+              handlers:
+                file:
+                  class: logging.FileHandler
```

Comment:
If we are logging by default, can we use rotating file handler as shown in the example below.

https://github.com/raintank/raintank-docker/blob/master/graphite-api/graphite-api.yaml#L22